extensions/intellij/intellij_generator_plugin/src/main/java/com/bloc/intellij_generator_plugin/action/BlocStatePackage.java

```diff
@@ -0,0 +1,11 @@
+package com.bloc.intellij_generator_plugin.action;
+
+/**
+ * @author Martin Neff, approppo GmbH (martin.neff@approppo.ch)
+ * @since 13.09.22
+ */
+public enum BlocStatePackage {
```

Comment:
```suggestion
public enum BlocTemplateType {
```

Reply:
thanks for the suggestion, ill refactor to the new name